tools/integration_tests/read_cache/cache_file_for_range_read_false_test.go

```diff
@@ -82,6 +82,10 @@ func (s *cacheFileForRangeReadFalseTest) TestRangeReadsWithCacheMiss(t *testing.
 }
 
 func (s *cacheFileForRangeReadFalseTest) TestConcurrentReads_ReadIsTreatedNonSequentialAfterFileIsRemovedFromCache(t *testing.T) {
+	if isParallelDownloadsEnabled(s.flags) {
+		// This test is not valid when Parallel Downloads are enabled.
```

Comment:
It was because we couldn't concretely say whether a read would be cache hit. I have instead enabled this test now and added the condition to cache Hit check